### PR TITLE
Initial script to experiment bamboo provisioning CFN stacks

### PR DESCRIPTION
**Pull request by amohammedalfresco**

### CODE REVIEW

**Comment by amohammedalfresco** [comment]
Assigning temporarily to myself until work is tested as expected.

File: cloudformation/ot-stack-createStack.sh [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  STACK_CREATION_TIMEOUT=600
+
+  # Essential Variables
+  export AWS_ACCESS_KEY_ID=$1
+  export AWS_SECRET_ACCESS_KEY=$2
+
+  # Some logging
+  printVarSummary
+
+  USER_NAME='admin'
```
Comment (SubatomicHero) [inline cloudformation/ot-stack-createStack.sh]: These need to be Bamboo variables passed in. As this repo is open source we cant have these passwords out in the wild.

File: cloudformation/ot-stack-deleteStack.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    STACK_NAME=$(echo -n $3| tr / - | awk '{print tolower($0)}')
+    printVarSummary
+
+    # Essential Variables
+    export AWS_ACCESS_KEY_ID=$1
+    export AWS_SECRET_ACCESS_KEY=$2
+
+    # Temp workaround until bug fix
+    separator
+    USAGE_PLAN_ID=$(aws cloudformation describe-stack-resources \
```
Comment (SubatomicHero) [inline cloudformation/ot-stack-deleteStack.sh]: The usage plan section isnt needed. Lines 21 - 27 can be removed

File: cloudformation/ot-stack-deleteStack.sh [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        --query "StackEvents[?ResourceType == 'AWS::CloudFormation::Stack'].ResourceStatus" \
+        --output text | head -n1)
+
+        if [[ "$STATUS" != "" ]]; then
+            monitorStatus $STATUS
+        else
+            logInfo "Stack successfully deleted"
+            # Part 2 of workaround
+            separator
+            logInfo "Deleting usage plan $USAGE_PLAN_ID"
```
Comment (SubatomicHero) [inline cloudformation/ot-stack-deleteStack.sh]: Lines 60 and 61 can be removed

**Pull request merged**